{{cookiecutter.project_slug}}/docker-compose.yml

```diff
@@ -1,6 +1,7 @@
 version: "3.1"
 services:
   web:
+    user: giraffe
```

Comment:
Won't this cause issues for developers using docker for mac?

Reply:
Honest answer: I don't know :smiley: since the volume mapping magic that Mac does should still work. I think it shouldn't pose a problem but I don't know for certain.